letsencrypt/acme/other.py

```diff
@@ -64,11 +109,10 @@ def verify(self, msg):
 
         """
         hashed = Crypto.Hash.SHA256.new(self.nonce + msg)
-        return Crypto.Signature.PKCS1_v1_5.new(self.jwk.key).verify(
-            hashed, self.sig)
+        return bool(Crypto.Signature.PKCS1_v1_5.new(self.jwk.key).verify(
```

Comment:
At some point of the development I was getting `1` instead of `True`, I'll recheck it and fix.

I've already implemented PSS in my local branch :)